src/main/java/mate/academy/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.service.impl;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.dao.ShoppingCartDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.Ticket;
+import mate.academy.model.User;
+import mate.academy.service.OrderService;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        List<Ticket> fromCart = shoppingCart.getTickets();
```

Comment:
According [Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/hibernate/add-order/add-order-hw)
```suggestion
        List<Ticket> fromCart = new ArrayList<>(shoppingCart.getTickets());
```